osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -65,56 +65,62 @@ protected override void PopOut()
 
         public void UpdateBeatmap(WorkingBeatmap beatmap)
         {
-            var lastContainer = beatmapInfoContainer;
-            float newDepth = lastContainer?.Depth + 1 ?? 0;
+            LoadComponentAsync(new BufferedWedgeInfo(beatmap)
+            {
+                Shear = -Shear,
+                Depth = info?.Depth + 1 ?? 0,
+            }, newInfo =>
+            {
+                // ensure we ourselves are visible if not already.
+                if (!IsPresent)

```

Comment:
This is a weird condition, if the BeatmapInfoWedge is not present, then I see no way that the BufferedWedgeInfo will ever get displayed. The previous code did not have this condition.

Reply:
is/was relying on AlwaysPresent above